app/src/docs/components/VisualEditorPage/Content.mdx

```diff
@@ -1,3 +1,637 @@
+import ScrollableAnchor from 'react-scrollable-anchor'
+import cx from 'classnames'
+import { Link } from 'react-router-dom'
+
+import CodeSnippet from '$shared/components/CodeSnippet'
+import { CreateStreamButton } from '$userpages/components/StreamPage/List'
+import { formatPath } from '$shared/utils/url'
+import links from '$shared/../links'
+
+import CreateStreamDialog from './images/streams/createStreamDialog.png'
+import EditStreamDetails from './images/streams/editStreamDetails.png'
+import MyFirstStreamViewPage from './images/streams/myFirstStreamViewPage.png'
+import ConfigureMyFirstStream from './images/streams/configureMyFirstStream.png'
+import CSVConfig from './images/streams/CSVConfig.png'
+import DropCSVHere from './images/streams/dropCSVHere.png'
+import SampleTwitterData from './images/streams/sampleTwitterData.png'
+import TwitterStreamView from './images/streams/twitterStreamView.png'
+import SubscribingToStream from './images/streams/subscribingToStream.png'
+
+import RoundToStep from './images/modules/roundToStep.png'
+import TablePopup from './images/modules/tablePopup.png'
+import TableModuleOptions from './images/modules/tableModuleOptions.png'
+import AndModule from './images/modules/andModule.png'
+import InitialValueDialog from './images/modules/initialValueDialog.png'
+
+import MyFirstStreamOnCanvas from './images/canvases/myFirstStreamOnCanvas.png'
+import BlankCanvasWithArrow from './images/canvases/blankCanvasWithArrow.png'
+import HideControlBarButton from './images/canvases/hideControlBarButton.png'
+import NewServiceWithArrow from './images/canvases/newServiceWithArrow.png'
+import OpenServiceWithArrow from './images/canvases/openServiceWithArrow.png'
+import SaveServiceWithArrow from './images/canvases/saveServiceWithArrow.png'
+import AddTwitterStream from './images/canvases/addTwitterStream.png'
+import ModuleBrowser from './images/canvases/moduleBrowser.png'
+import ConnectingStreamToModule from './images/canvases/connectingStreamToModule.png'
+import StartHistoricalRun from './images/canvases/startHistoricalRun.png'
+import PlaybackOptions from './images/canvases/playbackOptions.png'
+import StartRealtimeRun from './images/canvases/startRealtimeRun.png'
+import StopRealtimeRun from './images/canvases/stopRealtimeRun.png'
+
+import JavaModuleOnCanvas from './images/extensions/javaModuleOnCanvas.png'
+import JavaModuleExampleOnCanvas from './images/extensions/javaModuleExampleOnCanvas.png'
```

Comment:
The actual file name begins with capital `J`, got an error for that while building.